### PR TITLE
OLD BRANCH - Feature/notes router+model

### PR DESCRIPTION
**Pull request by thecuriousones**
# Description
Loon Video: [](https://www.loom.com/share/fdcdae69f53b486286e548952d2eefeb)
This pull request contains the model and router files for the client notes system.

Fixes #

- What work was done?
      - Model and router files were created
- Why was this work done?
      - Frontend has a way to access the backend's database containing mock client notes for the notes system
- What feature / user story is it for?
      -  A system containing all of the case managers' notes for different clients (households)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [ ] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

## Has This Been Tested

- [ ] Yes
- [x] No
- [ ] Not necessary

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by ashtilawat — COMMENTED** [review]
Just want to flag the 12000 lines deleted from the package-lock. Just want to make sure that was planned. If so, what package was changed/updated?

**Comment by thecuriousones** [comment]
This is an old branch actually I had to make a new one cause it wouldn't let me merge or push my commits. I actually need to make new pr with my new branch but it will have all the same information minus the placement of the routes and models. As for the 12000 lines deleted as soon as I did my first commit on this old branch for some reason it did that I was going to come to you for help but since I had to make a whole new branch it no longer mattered.

File: api/clientnotes/notesModel.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
+const db = require('../../data/db-config');
```
Comment (stepbystepdayo) [inline api/clientnotes/notesModel.js:1]: I think all this code is in the other branch, right? Looks great!

**Review by stepbystepdayo — COMMENTED** [review]
Thank you for all the work Tamara! I will keep looking at your other branch, that is current.

**Pull request closed without merging**